.github/workflows/deploy-book.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Copy and rename the PDF to the HTML dir
         run: |
           cp src/_pdfbuild/_build/pdf/book.pdf \
-          src/_build/html/software-gardening-almanack.pdf
+          src/book/_build/html/software-gardening-almanack.pdf
```

Comment:
Curious if you know why this happened to break randomly? Could it be due to a change in the working directory that now requires a more specific path?

Reply:
Thanks, good question - I mistakenly forgot the parent directory path when adding this copy statement the first time.